packages/modeling/src/operations/modifiers/comparePolygons.js

```diff
@@ -0,0 +1,10 @@
+const comparePolygon = require('../../../test/helpers/comparePolygons') // rename
```

Comment:
Is this production code relying on test code?

Reply:
UG. I forgot. I renamed and moved to test/helpers.

DONE